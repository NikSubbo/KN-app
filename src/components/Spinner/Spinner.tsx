import { FC } from 'react';

export const Spinner: FC = () => (
  <div
    className="min-h-screen flex justify-center items-center"
    data-testid="spinner"
  >
    <div className="animate-spin rounded-full h-32 w-32 border-b-2 border-aqua"></div>
  </div>
);

export default Spinner;
