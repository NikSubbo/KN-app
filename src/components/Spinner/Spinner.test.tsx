import { render } from '@testing-library/react';
import Spinner from './Spinner';

describe('Spinner', () => {
  it('should match the snapshot', () => {
    const { container } = render(<Spinner />);
    expect(container).toMatchSnapshot();
  });
});
