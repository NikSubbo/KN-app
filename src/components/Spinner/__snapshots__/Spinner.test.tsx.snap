// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should match the snapshot 1`] = `
<div>
  <div
    class="min-h-screen flex justify-center items-center"
    data-testid="spinner"
  >
    <div
      class="animate-spin rounded-full h-32 w-32 border-b-2 border-aqua"
    />
  </div>
</div>
`;
